crates/pevm/Cargo.toml

```diff
@@ -0,0 +1,76 @@
+[package]
+name = "pevm"
+version = "0.1.0"
+edition = "2021"
+
+[features]
+defaults = []
+rpc = [
```

Comment:
Oh one last thing but it should be `rpc-storage` since `rpc` is very broad and can be misleading.